manifests/config.pp

```diff
@@ -31,14 +31,35 @@
   }
 
   # Enable or disable ODL OVSDB ML2 L3 forwarding
+  $custom_properties = '/opt/opendaylight/etc/custom.properties'
   file { 'custom.properties':
     ensure  => file,
-    path    => '/opt/opendaylight/etc/custom.properties',
+    path    => $custom_properties,
     # Set user:group owners
     owner   => 'odl',
     group   => 'odl',
-    # Use a template to populate the content
-    content => template('opendaylight/custom.properties.erb'),
+  }
+  if $opendaylight::enable_l3 == 'yes' or $opendaylight::enable_l3 == true {
```

Comment:
This actually doesnt do anything anymore with new netvirt.  We can remove the enable_l3 parameter.  We should actually cut a stable/boron branch, and you can remove this on master.